apps/vc-api/docker-compose.dev.yaml

```diff
@@ -0,0 +1,22 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:14-alpine
+    restart: unless-stopped
+    environment:
+      POSTGRES_PASSWORD: postgres
```

Comment:
I thnk there must be single source of truth of config. Can we reference `.env` here